lib/Products/Product.hs

```diff
@@ -1,17 +1,25 @@
-{-# LANGUAGE OverloadedStrings #-}
-
 module Products.Product where
-  import Control.Monad.Trans.Class (MonadTrans)
-  import Control.Monad.IO.Class (MonadIO)
-  import Database
-  import qualified Database.Persist as DB
+  import qualified Data.Text as T
+  import Database (runDB)
   import qualified Database.Persist.Postgresql as DB
   import GHC.Int (Int64)
   import Models
 
+  data Product = Product { productName :: T.Text 
+                         } deriving (Show)
+
```

Comment:
Yeah, it could be seen as YAGNI right now, but I felt like I wanted to create a conceptual space between the persisted data model and the data type the module exposes to the world. I'm hoping that introducing the Product model will allow us to create a useful interface for the module.